realsense2_camera/CMakeLists.txt

```diff
@@ -115,7 +115,10 @@ find_package(nav_msgs REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(diagnostic_updater REQUIRED)
-find_package(OpenCV REQUIRED COMPONENTS core)
+
+if(WIN32)
```

Comment:
If OpenCV is indeed needed let's install it always.

Sorry for the confusion.
Let's remove all the new `if(WIN32)`

Reply:
done